operations/workplace/people/performance-reviews-50/formal-review-process.md

```diff
@@ -1,13 +1,47 @@
-4. **Peer feedback**: Your selected peers will receive a request to complete the [annual review feedback survey](https://docs.google.com/forms/d/1JNlDCAOgUjjLSJj5EoNe7Pl5kqN6KN7LaEIlM5L9LXw/edit) within two weeks of receipt. On average, teammates should be completing feedback surveys for 3 to 6 of their peers. 
-5. **Self reflection**: Each staff member will receive a BambooHR notification via email to complete their Self-Assessment in BambooHR. 
-6. **Draft of annual review**: Once a staff member completes their Self-Assessment in BambooHR, Managers will be notified. The Self-Assessment and Manager Review Forms are combined in one document in BambooHR. Manager will read all feedback from the surveys and incorporate their feedback based on their own performance expectations of this role. Managers can draft performance reviews. 
-**Note**: Once a Manager completes the review via their signature in BambooHR, the feedback form will be automatically shared with the direct report. It is helpful to share the review ahead of the 1:1 meeting to give an opportunity to read feedback, but please plan timing accordingly.  
-9. **Calibration**: Managers meet to review and discuss feedback and assessments to ensure consistency in evaluation against the criteria for your role and level. The Team MLT will decide how to group managers for calibration. Typically this is done by function. For example, G&A Managers will attend one group calibration meeting. HR will share a sheet with all staff members ahead of the calibration. Managers will be asked to make note of any level change requests and performance review ratings (meets, does not meet, exceeds) for each direct report.
-10. **Review discussion**: You and your Manager will meet to discuss your achievements in the past review period, the peer, self, and manager feedback, any compensation changes recommended by HR, and opportunities to further grow and develop.
+## Confirm level expectations for roles are up to date
+
+If any level descriptions need to be updated or written, Managers complete it before the review cycle begins.
+
+## Manager training
+
+Before each review cycle, managers read the performance review chapter in High Output Management. Managers will be invited to attend [Manager Training on Performance Management at Mattermost](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#manager-training) at the beginning of the [Annual Performance Review cycle](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#what-is-the-process). If you don't have a copy, use [this form](https://forms.gle/SUWWvZZGqg5B4d1aA) to request a copy.
+
+## Feedback panel requests
+
+Managers will provide peers for each direct report who can most effectively provide feedback on achievements and performance over the past year \(okay to include community members in addition to staff\).
```

Comment:
```suggestion
Managers will provide peers for each direct report who can most effectively provide feedback on achievements and performance over the past year \(this can include community members in addition to staff\).
```